app/controllers/DeparturesController.scala

```diff
@@ -89,8 +89,13 @@ class DeparturesController @Inject()(
                         case submissionFailureRejected: SubmissionProcessingResult.SubmissionFailureRejected =>
                           BadRequest(submissionFailureRejected.responseBody)
                         case SubmissionProcessingResult.SubmissionSuccess =>
-                          auditService.auditEvent(DepartureDeclarationSubmitted, request.enrolmentId, departure.messages.head, request.channel)
-                          auditService.auditEvent(MesSenMES3Added, request.enrolmentId, departure.messages.head, request.channel)
+                          val len = request.headers.get(play.api.http.HeaderNames.CONTENT_LENGTH).get.toInt
```

Comment:
we can probably move this into our request case class to make it a little cleaner to read here.